pkg/gatherers/conditional/conditional_gatherer.go

```diff
@@ -128,6 +132,39 @@ var defaultGatheringRules = []GatheringRule{
 			},
 		},
 	},
+	// AlertManagerLogs
+	{
+		Conditions: []ConditionWithParams{
+			{
+				Type: AlertIsFiring,
+				Alert: &AlertConditionParams{
+					Name: "AlertmanagerClusterFailedToSendAlerts",
+				},
+			},
+		},
+		GatheringFunctions: GatheringFunctions{
+			GatherAlertmanagerLogs: GatherAlertmanagerLogsParams{
+				AlertName: "AlertmanagerClusterFailedToSendAlerts",
+				TailLines: 50,
+			},
+		},
+	},
```

Comment:
The first condition seems a bit unnecessary IMO. The problem is that the `AlertmanagerClusterFailedToSendAlerts` doesn't provide any "pod" info and thus it always fails at https://github.com/openshift/insights-operator/pull/545/files#diff-9a6e0f2be79b4492a5ed592bd137bcb21a989988ccfc185f16796c550ceedd3fR70 and we also have https://github.com/openshift/insights-operator/blob/master/pkg/gatherers/conditional/conditional_gatherer.go#L133 (not sure why) so I am not sure about all consequences....

Reply:
Let me run some tests, but I think that you are right, we may not need the first alert.

Reply:
I ran some tests and read some docs, and from what I can say I didn't notice any issues to remove the first alert and keep just the second one.